### PR TITLE
Add mongoquery-v1.1.0

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@kapouille I am adding mongoquery to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge mongoquery). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/mongoquery-feedstock shortly after this PR is finished and merged. 

If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!

File: recipes/mongoquery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    imports:
+        - mongoquery
+    requires:
+        - pytest
+    commands:
+        - py.test {{ environ.SRC_DIR }}/tests -vrsx
+
+about:
+    home: https://github.com/kapouille/mongoquery
+    license: The Unlicense
```
Comment (ericdill) [inline recipes/mongoquery/meta.yaml]: @ocefpaf @jakirkham Is this the way you would like to refer this license?

[Here] is the license that accompanies the git repo

Comment (jakirkham) [inline recipes/mongoquery/meta.yaml]: Personally, I would prefer it to say something like "Public Domain". Everyone knows what that means and we use that in a few cases.

File: recipes/mongoquery/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        intended to parse objects structured as fundamental types in
+        a similar fashion to what is produced by JSON or YAML
+        parsers.
+
+extra:
+    recipe-maintainers:
+        - ericdill
+        - licode
+        - tacaswell
+
```
Comment (jakirkham) [inline recipes/mongoquery/meta.yaml]: nit: please remove extra newline.

**Comment by kapouille** [comment]
@ericdill 
First time I'm hearing about Conda. Sounds like I'd definitely need to read up on this. I'll be happy to be added to the list of maintainers, although I probably need to first get up to speed.

**Comment by ericdill** [comment]
@jakirkham Thanks for the review. I've made the suggested changes.

@kapouille Thanks for responding. I've added you as a maintainer. We are all happy to help you get up to speed on conda. Let us know how we can help!

**Comment by ocefpaf** [comment]
LGTM

**Comment by ericdill** [comment]
Thanks @ocefpaf!

**Pull request merged**